docs/versions/latest/Release.md

```diff
@@ -120,3 +126,9 @@ You don't need to migrate projects if they are `abx` based. `nodejs` is the corr
 ### *Migrate PolicyTemplates*
 
 Search your projects that use `@PolicyTemplate` decorator. The `templateVersion` property is now required. Check on top for possible values
+
+### *Migrate away from `vrang.project.id`*
```

Comment:
We might need a similar section for `vrang_org_id` and `installer` file, e.g. `env.properties`

Reply:
Will open a new PR